examples/rust/candid.did

```diff
@@ -16,10 +16,13 @@ type Utxo = record {
   confirmations: nat32;
 };
 
+type BTCCanisterInfo = record {
```

Comment:
```suggestion
type BtcCanisterInfo = record {
```

Reply:
What is this record used for?
"BTC canister info" sounds like it provides information about the (state of the) Bitcoin canister.

Reply:
Okay, having read the rest of the proposed changes, it is the single response that is now provided. As stated below, I think we should keep the functionality as is. The goal is to simplify the code only.